src/TLuaInterpreter.cpp

```diff
@@ -6634,6 +6642,13 @@ int TLuaInterpreter::isActive(lua_State* L)
             }
             it1++;
         }
+    } else if (type.compare(QLatin1String("script"), Qt::CaseInsensitive) == 0) {
```

Comment:
Like the previous comments - this is also in #2195.

To fix the duplication please change it back **in this PR**...